keras/engine/saving.py

```diff
@@ -4,17 +4,21 @@
 from __future__ import absolute_import
 from __future__ import division
 
+
```

Comment:
nit: I probably wouldn't add a second blank line here, since all the import groups are separated by a single line elsewhere.